ballerina/tests/resources/expected_results/enum_type_default_value_with_string_literal.json

```diff
@@ -1,7 +1,7 @@
 {
     "errors": [
         {
-            "message": "Variable \"day\" of type \"Weekday\" has invalid default value: MONDAY. Expected type \"Weekday\", found MONDAY",
+            "message": "Variable day expected value of type \"Weekday\", found MONDAY",
```

Comment:
Shouldn't we have double quotes for the variable name? Is the double quotes around the type name necessary?

Reply:
both are needed.